egs/ljspeech/tts1/local/ob_eval/evaluate_cer.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+# Copyright 2019 Okayama University (Katsuki Inoue)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+echo "$0 $*"  # Print the command line for logging
+. ./path.sh
+
+nj=1
+do_delta=false
+eval_tts_model=1
+db_root=""
+voc="GL"
+backend=pytorch
+api=v1
+help_message="Usage: $0 <asr_model> <outdir> <subset>"
+
+. utils/parse_options.sh
+
+asr_model=$1
+outdir=$2
+name=$3
+
+if [ $# != 3 ]; then
+    echo "${help_message}"
+    exit 1;
+fi
+
+set -euo pipefail
+
+echo "stage 6.0: Model preparation"
+# ASR model selection for CER objective evaluation 
+# Please add new model if you want to use your ASR model.
+asr_model_dir="exp/${asr_model}_asr"
+case "${asr_model}" in
+    "librispeech.transformer.ngpu4") asr_url="https://drive.google.com/open?id=1BtQvAnsFvVi-dp_qsaFP7n4A_5cwnlR6" \
+      asr_cmvn="${asr_model_dir}/data/train_960/cmvn.ark" \
+      asr_pre_decode_config="${asr_model_dir}/conf/tuning/decode_pytorch_transformer_large.yaml" \
+      recog_model="${asr_model_dir}/exp/train_960_pytorch_train_pytorch_transformer.v1_aheads8_batch-bins15000000_specaug/results/model.val5.avg.best" \
+      lang_model="${asr_model_dir}/exp/irielm.ep11.last5.avg/rnnlm.model.best" ;;
+
+    *) echo "No such models: ${asr_model}"; exit 1 ;;
+esac
+
+# ASR model download (librispeech)
+if [ ! -e ${asr_model_dir}/.complete ]; then
+    mkdir -p ${asr_model_dir}
+    download_from_google_drive.sh ${asr_url} ${asr_model_dir} ".tar.gz"
+    touch ${asr_model_dir}/.complete
+fi
+echo "ASR model: ${asr_model_dir} exits."
+
+# Select TTS model
+if [ ${eval_tts_model} == 1 ]; then
+    echo "Evaluate: TTS model"
+else
+    echo "Evaluate: ground truth"
+    expdir=exp/ground_truth
+    outdir=${expdir}/sym_link
+    
+    if [ ${db_root} == "" ]; then
+        echo "Please set --db_root"; exit 1
+    fi
+    mkdir -p ${outdir}_denorm/${name}/wav
+    cat < data/${name}/wav.scp | awk '{print $1}' | while read -r filename; do
+        if [ -L ${outdir}_denorm/${name}/wav/${filename}.wav ]; then
+            unlink ${outdir}_denorm/${name}/wav/${filename}.wav
+        fi
+        ln -s ${db_root}/LJSpeech-1.1/wavs/${filename}.wav ${outdir}_denorm/${name}/wav/${filename}.wav
+    done
+fi
+
+# Select decoder
+if [ ${voc} == "GL" ]; then
+    dir_tail=""
+elif [ ${voc} == "WNV_softmax" ]; then
+    dir_tail="_wnv_nsf"
+elif [ ${voc} == "WNV_mol" ]; then
+    dir_tail="_wnv_mol"
+fi
```

Comment:
I guess this is what you have locally, right? Since wavenet vocoder generation hasn't been incorporated in the recipe (pretty straightforward though https://github.com/r9y9/espnet/blob/cce17d48caa80786bf222c466b58daf2754b51d9/egs/jsut/tts1/run.sh#L241-L265), how about removing them for now?

Reply:
This code is a local setting. I remove them for now.